### ISSUE TITLE
Using 'Since=' for fetch_ohlcv() not working consistently with python

### ISSUE THREAD

**Issue by topchatz** [ccxt/ccxt#5695]
- OS: Mac Sierra 10.12.3
- Programming Language version: Python 3.7.0
- CCXT version: 1.18.1060
- Exchange: KuCoin
- Method: fetch_ohlcv()

Hi guys, this is the first time I've ever used GitHub so apologies if I've not done something correctly.

I'm a big, big fan of ccxt, but I have a question about fetch_ohlcv() using Python.

I'm trying to get ohlcv data going back further that the default 1000 data point limit for various exchanges.

To do this I do, for example:

```
exchange = ccxt.binance()
day = '2019-07-26 16:45:00'
since = round(datetime.strptime(str(day), '%Y-%m-%d %H:%M:%S').timestamp()*1000)
df = pd.DataFrame(exchange.fetch_ohlcv('ADA/USDT', '15m', since=since, limit=1000))
```
I would then expect to get 1000 data points starting from 2019-07-26 16:45:00. This works perfectly for the likes of Binance or HitBTC2 (although not HitBTC), however it does't work for the likes of Kucoin (either Kucoin or Kucoin2). I wondered if there was anything I could do about it or does Kucoin actively block this?

Many thanks

**Comment by kroitor** [comment]
@topchatz thx! This is fixed in 1.18.1062 (will arrive in 15 minutes or so). We will be happy if you report back if it solves the issue for you.

**Comment by topchatz** [comment]
> @topchatz thx! This is fixed in 1.18.1062 (will arrive in 15 minutes or so). We will be happy if you report back if it solves the issue for you.

Hi @kroitor , amazing response time, thanks! I can confirm the update is working! Or it does for the exchanges I noticed weren't working. So it now works with both Kucoin and Kucoin2 as well as hitbtc2, however hitbtc doesn't work, it only returns 21 data points. However, I use hitbtc2 by default now anyway so it's not a problem, just thought it was worth a mention. Thanks again mate.
